pyof/utils.py

```
@@ -61,3 +65,20 @@ def unpack(packet):
         return message
     except (UnpackException, ValueError) as exception:
         raise UnpackException(exception)
+
+
+def is_ofbac_bad_out_port(code):
+    """Check if the code is a OFPBAC_BAD_OUT_PORT."""
+    return code in (BadActionCode01.OFPBAC_BAD_OUT_PORT,
+                    BadActionCode04.OFPBAC_BAD_OUT_PORT)
+
+
+def get_port_config_for_version(version):
+    """Return port_config object to a specific version of python-openflow."""
+    port_config = None
+
+    if version == 0x01:
+        port_config = PortConfig01.OFPPC_NO_FWD
+    elif version == 0x04:
+        port_config = PortConfig04.OFPPC_NO_FWD
```

```suggestion
    if version == 0x01:
        port_config = v0x01.common.phy_port.PortConfig.OFPPC_NO_FWD
    elif version == 0x04:
        port_config = v0x04.common.phy_port.PortConfig.OFPPC_NO_FWD
```

---

Also, I think it's better to return an error if the version is not supported.